tsweb/tsweb.go

```diff
@@ -396,6 +397,29 @@ func VarzHandler(w http.ResponseWriter, r *http.Request) {
 				dump(name+"_", kv)
 			})
 			return
+		case PrometheusMetricsReflectRooter:
+			root := v.PrometheusMetricsReflectRoot()
+			rv := reflect.ValueOf(root)
+			if rv.Type().Kind() == reflect.Ptr {
+				rv = rv.Elem()
```

Comment:
and if rv is nil?

Reply:
PTAL; more cases & tests, including that.